src/rules/order.js

```diff
@@ -623,7 +626,7 @@ module.exports = {
           name = node.moduleReference.expression.value
           type = 'import'
         } else {
-          name = context.getSourceCode().getText(node.moduleReference)
+          name = '',
```

Comment:
should this use the `nameResolver` logic perhaps?

Reply:
Not quite. The name is changed to `''` here in order to "disable" alphabetization for this kind of import.
Truth be told: This is a pretty dirty workaround, so I'm sorry 'bout this weird change 😅

Another approach would be to add a `value` and `displayName` property to the `imported`-entries in place of the `name` property.

This way we could set `value` to the... well... value to be sorted on and `displayName` the name to be shown in error-messages.

What do you think about this?

Reply:
I think that sounds more explicit and thus better :-)

Reply:
Alright, I fixed it now 😄